scenes/wifi_marauder_scene_start.c

```diff
@@ -68,6 +75,13 @@ const WifiMarauderItem items[NUM_MENU_ITEMS] = {
      NO_ARGS,
      FOCUS_CONSOLE_END,
      SHOW_STOPSCAN_TIP},
+    {"Evil Portal",
+     {"Start"},
```

Comment:
Can you make "start" lowercase for consistency?

Note: after starting the Evil Portal, I pressed the back button (stopscan). This message was logged: "E (149829) wifi_init: Failed to deinit". But it still seems to be fine?

Reply:
Fixed formatting. The message likely popped up because the WiFi never initialized when you executed the Evil Portal attack. This happens if Evil Portal fails to start due to there not being an AP config or HTML file to use. If those steps fail, it doesn't even bother to initialize the WiFi. When you hit the back button and initiate "stopscan", it tries to de-initialize an uninitialized WiFi. It will still work just fine and I would still recommend using stopscan even if the Evil Portal config init fails.